account_reconciliation_widget/static/src/xml/account_reconciliation.xml

```diff
@@ -532,7 +532,7 @@
                     /> <t t-esc="line.account_name" /></td></tr>
         <tr><td>Date</td><td><t t-esc="line.date" /></td></tr>
         <tr><td>Due Date</td><td><t t-esc="line.date_maturity || line.date" /></td></tr>
-        <tr><td>Journal</td><td><t t-esc="line.journal_id.display_name" /></td></tr>
+        <tr><td>Journal</td><td><t t-esc="line.journal_name" /></td></tr>
```

Comment:
Shouldn't it be and don't populate the other?

```suggestion
        <tr><td>Journal</td><td><t t-esc="line.journal_id[1]" /></td></tr>
```

Reply:
Ok, I agree with you. Already Updated.